### PR TITLE
Rm speed CDF, replaced w/ speed Distribution

### PR DESCRIPTION
**Pull request by youralien**
Summary:

- We were currently making CDF / PDF distribution visualizations with
  each call of speedCDF.
- Now, we only make the PDF distribution
- Since CDF was removed, I've renamed it more aptly to "Distribution"
  so that the routes, handlers, and methods are no longer a misnomer

### CODE REVIEW

**Comment by youralien** [comment]
Depends on this PR in GUI: https://github.com/santosfamilyfoundation/SantosGUI/pull/60

**Comment by youralien** [comment]
BTW will have to update this with API docs before merging

File: app/handlers/createSpeedDistribution.py [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
+class CreateSpeedDistributionHandler(BaseHandler):
     """
-    @api {post} /speedCDF/ Speed CDF
-    @apiName SpeedCDF
+    @api {post} /speedDistribution/ Speed Distribution
+    @apiName SpeedDistribution
     @apiVersion 0.1.0
     @apiGroup Results
-    @apiDescription Calling this route will create a graph of the speed CDF's from a specified project.
+    @apiDescription Calling this route will create a graph of the speed Distribution's from a specified project.
```
Comment (runnersaw) [inline app/handlers/createSpeedDistribution.py]: No caps

File: app/handlers/createSpeedDistribution.py [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
 
-    @apiParam {String} identifier The identifier of the project to create a speed CDF for.
+    @apiParam {String} identifier The identifier of the project to create a speed Distribution for.
```
Comment (runnersaw) [inline app/handlers/createSpeedDistribution.py]: no caps

**Review by runnersaw — APPROVED** [review]
:ship: :it:

**Pull request merged**